### PR TITLE
Graph to swc

### PR DESCRIPTION
**Pull request by alishakodibagkar**

### CODE REVIEW

**Comment by bvarjavand** [comment]
Great. Next are typing, docstrings, and tests. Pull from `develop` and look at `brainlit/utils/upload.py` and `tests/test_upload.py` for reference.

File: brainlit/utils/swc.py [hunk 1]
```diff
@@ -515,3 +518,49 @@ def swc2skeleton(swc_file, origin=None):
...
+    """
+    Generates a *.swc file from a networkx graph.
+    Parameters
+    ----------
+    G : :object: networkx graph object
+    resolution : list
+        Of the format [x_res, y_res, z_res]
+    bbox : :object: Bounding box, optional (default=None)
+    fpath : str, optional (default="demo.swc")
+    """
```
Comment (tathey1) [inline brainlit/utils/swc.py:531]: did you write this code yourself or take it from somewhere else?

File: brainlit/utils/swc.py [hunk 2]
```diff
@@ -515,3 +518,49 @@ def swc2skeleton(swc_file, origin=None):
...
+    res = ngl_sess.cv_segments.scales[0]["resolution"]
```
Comment (tathey1) [inline brainlit/utils/swc.py:532]: where does the ngl_sess object come from?

**Comment by tathey1** [comment]
Is there any reason to keep this open @alishakodibagkar

**Comment by alishakodibagkar** [comment]
no need to keep this open

**Pull request closed without merging**